LayoutTests/fast/editing/editing-position-crash.html

```diff
@@ -21,7 +21,7 @@
 </head>
 <body>
 <svg onfocusin="insert()">
-<path id="path0" tabindex="2" onfocusin="window.getSelection().setPosition(obj0, 10);" />
+<path id="path0" tabindex="2" onfocusin="window.getSelection().setPosition(obj0, 1);" />
```

Comment:
Surprised you did not land this change separately

Reply:
Yeah, I noticed I forgot to split 3-4 test changes after updating the change log. We could split into their own PRs if you'd like.

Reply:
Does not seem necessary if this is ready to go. If we end up having to revert this for some reason would be nice to have it be as small as possible.